includes/manage_pages.inc

```diff
@@ -897,3 +913,107 @@ function islandora_paged_content_manage_page_progression_form_submit($form, &$fo
   islandora_paged_content_set_page_progression($object, $form_state['values']['page_progression']);
   drupal_set_message(t('Page progression updated.'));
 }
+
+/**
+ * Form handler for setting the Viewing mode hint for a paged content.
+ *
+ * @param array $form
+ *   The Drupal form.
+ * @param array $form_state
+ *   The Drupal form state.
+ * @param AbstractObject $object
+ *   The object to fetch the pages from.
+ *
+ * @return array
+ *   The Drupal form.
+ */
+function islandora_paged_content_manage_viewing_hint_form(array $form, array &$form_state, AbstractObject $object) {
+  module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
+  $viewing_hint = islandora_paged_content_get_viewing_hint($object);
+  $viewing_hint = isset($viewing_hint) ? $viewing_hint : 'notset';
+  $form_state['pid'] = $object->id;
+  return array(
+    'viewing_hint' => array(
+      '#type' => 'radios',
+      '#title' => t('Viewing Mode Hint'),
+      '#description' => t("This affects how Islandora viewers display page sequences. 'Paged' here is equivalent to the '2-up' mode in IA Book Reader. If left as 'Not set', global option for each viewer will be respected."),
+      '#options' => array(
+        'notset' => t('Not set'),
```

Comment:
As @jonathangreen suggests... should change to "System default"

Reply:
@bondjimbond ok, will deal with this ASAP, thanks!